_includes/manuals/nightly/3.3.1_rpm_packages.md

```diff
@@ -85,7 +82,7 @@ Install foreman and other foreman-* packages to add functionality:
 
 1. Configure by editing `/etc/foreman/settings.yaml` and `/etc/foreman/database.yml`
 1. After changing the database, migrate it: `sudo -u foreman /usr/share/foreman/extras/dbmigrate`
-1. Start the foreman service or set up passenger: `service foreman start`
+1. Start the foreman service or set up passenger: `systemctl start foreman`
```

Comment:
This needs the foreman-service package to be installed. Will need a larger overhaul.